### ISSUE TITLE
getting 84% in ucf101 on split 1

### ISSUE THREAD

**Issue by dandingol03** [yjxiong/tsn-pytorch#125]
Hi @yjxiong .
 Thanks for your amazing work! I run the training on the google colab and limit the batch-size to 64, i use bn-inception. But i get 84% accuracy finally. I was wondering is the batch-size setting makes the accuracy so different?

**Comment by dingtao1** [comment]
me too, have you solved the problem?

**Comment by dandingol03** [comment]
i guess is the batch-size makes the accuracy lower
